app/controllers/application_controller.rb

```diff
@@ -1,5 +1,20 @@
 class ApplicationController < ActionController::Base
 
+  def self.load_image
+    #where
+    conn = Faraday.new(url: "https://api.unsplash.com/") do |faraday|
+      faraday.params['client_id'] = ENV['UNSPLASH_API']
+    end
+
+    #ENV['UNSPLASH_API'] will tell the thing to use my key in the application.yml
+    #what
+    response = conn.get("photos/frlKR1JfDok")
+
+    json = JSON.parse(response.body, symbolize_names: true)
+
+    @photo = Photo.new(json)
+  end
+
```

Comment:
line by line fo the method 


5:access the  API from the declared URL
6: is basically doing a query param(see important notes for the UNSPLASH_API)
the docs have it written as "https://api.unsplash.com/photos/?client_id=YOUR_ACCESS_KEY" 

11 :is getting what you want. the URL in the get is the link to the logo we chose earlier(basically go to google and get everything after the .com/) 
"https://api.unsplash.com/photos/frlKR1JfDok"

13 is making the data readable(well making it into a big hash)

15:creating a photo object with the initial argument of the big hash